types/aws-lambda/trigger/eventbridge.d.ts

```diff
@@ -0,0 +1,32 @@
+import { Handler } from '../handler';
+
+export type EventBridgeHandler<TDetail = any, TResult = any> = Handler<EventBridgeEvent<TDetail>, TResult>;
+
+export type JSONPrimitive = string | number | boolean | null;
+// this could be defined better taking advantage of recursive types, we kept it simple to avoid to increse the required Typescript version
+export type JSONValue = JSONPrimitive | JSONObject | JSONPrimitive[] | JSONObject[];
+
+export interface JSONObject {
+    [member: string]: JSONValue;
+}
+
+// credits to: https://github.com/microsoft/TypeScript/issues/1897#issuecomment-580962081
+export type JSONCompatible<T> = {
+    [P in keyof T]: T[P] extends JSONValue
+        ? T[P]
+        : Pick<T, P> extends Required<Pick<T, P>>
+        ? never
+        : JSONCompatible<T[P]>;
+};
```

Comment:
Ok, the point was to prevent issues like someone usa a Date and get troubles only on runtime. If we decide that it is not so important I can remove it. Let me know.

Reply:
Just removed